soc/arm/st_stm32/stm32l4/power.c

```diff
@@ -16,6 +16,12 @@
 #include <logging/log.h>
 LOG_MODULE_DECLARE(soc, CONFIG_SOC_LOG_LEVEL);
 
+#if defined(CONFIG_CLOCK_STM32_SYSCLK_SRC_MSI)
```

Comment:
comment is also possible here 
`/* select MSI as wake-up system clock if configured, HSI otherwise */`

Reply:
fixed, thanks